StreamChatClient_v3/Workers/Background/ChannelEventsHandler.swift

```diff
@@ -8,7 +8,7 @@ import CoreData
 class ChannelEventsHandler<ExtraData: ExtraDataTypes>: EventHandlerWorker<ExtraData> {
     override func handleNewEvent(event: Event) {
         // TODO: Saving the data must happen in the middleware so we're sure the data is saved before the event is published
-//    if let event = event as? AddedToChannel<ExtraData> {
+//    if let event = event as? NotificationAddedToChannel<ExtraData> {
```

Comment:
It's great you also took time to update unsued code 😂

Reply:
I think it was automatically with renaming.